services/slam/fake/slam_test.go

```diff
@@ -267,12 +116,10 @@ func reverse[T any](slice []T) []T {
 	return slice
 }
 
-func verifyGetPointCloudMapStreamStateful(t *testing.T, slamSvc *SLAM, extra map[string]interface{}) {
+func verifyGetPointCloudMapStateful(t *testing.T, slamSvc *SLAM) {
 	testDataCount := maxDataCount
 	getPointCloudMapResults := []float64{}
```

Comment:
just curious why both the old and new api's were using the same variable for map but different variables for position and internal state?

Reply:
missed this/forgot while going thru the code with you, this PR also removes `verifyGetMapStateful`, which tested the original getMap endpoint.